format/Schema.fbs

```diff
@@ -255,6 +255,10 @@ enum IntervalUnit: short { YEAR_MONTH, DAY_TIME}
 // DAY_TIME - Indicates the number of elapsed days and milliseconds,
 //   stored as 2 contiguous 32-bit integers (8-bytes in total).  Support
 //   of this IntervalUnit is not required for full arrow compatibility.
+// MONTH_DAY_NANOS - A triple of (# of months, # of days, # nanoseconds).
+//  The values are stored contiguously in 16 byte blocks. Months and
+//  days are encoded as 32 bit integers and nanoseconds is encoded as a
+//  64 bit integer.
```

Comment:
Indicate endianness (this should be done for the other unit types too)?

Reply:
yes.  Added a sentence at the top.  I believe these all use endianness indicated by the schema.